src/startup.js

```diff
@@ -7,6 +7,8 @@ import seedEmailTemplatesForShop from "./util/seedEmailTemplatesForShop.js";
  * @returns {undefined}
  */
 export default async function emailTemplatesStartup(context) {
+  await seedEmailTemplatesForShop(context, "kxwJD6EKFKkNrBjtJ");
```

Comment:
The shop ID here shouldn't be hardcoded. Can you get it from context or an additional param?

Reply:
Damn, I keep on forgetting this piece of debugging code. Second time in a row it happens to me. Sorry!